### ISSUE TITLE
 Gdx.input.setCursorCatched(true) crashes app on macOs with M1 chip

### ISSUE THREAD

**Issue by PilzHere** [libgdx/libgdx#6989]
#### Issue details
Calling this particular function on macOs with M1 chip crashes the app:
```java
Gdx.input.setCursorCatched(true);
```

#### Reproduction steps/code
1. Use macOs with M1
2. Create new LibGdx 1.11.0 project
3. Add dependency:
```gradle
api "com.badlogicgames.gdx:gdx-lwjgl3-glfw-awt-macos:$gdxVersion"
```
4. Set source compatability to 1.8 in ``build.gradle``
5. Call function:
```java
Gdx.input.setCursorCatched(true);
```

#### Version of libGDX and/or relevant dependencies
* macOs Monterey
* JDK 17
* LibGdx 1.11.0 + gdx-lwjgl3-glfw-awt-macos

#### Stacktrace
[link](https://gist.github.com/PilzHere/5fe1fb8cd309f65f9a3ce030c551732c)
#### Content from ``hs_err_pid28244.log``
[link](https://gist.github.com/PilzHere/cdc31f31cd5ccf8a4d1d15c391e0d0c8)

#### Please select the affected platforms
- [ ] Android
- [ ] iOS
- [ ] HTML/GWT
- [ ] Windows
- [ ] Linux
- [x] macOS

**Comment by soundasleep** [comment]
Could this be related to glfw not supporting captured cursor mode on Cocoa? see: https://github.com/glfw/glfw/issues/58

**Comment by crykn** [comment]
Unlikely. This is caused by the `gdx-lwjgl3-glfw-awt-macos` extension and would be fixed by #7361.
